01-data-structures/06-trees/min_tree/min_tree_spec.rb

```diff
@@ -0,0 +1,29 @@
+include RSpec
+
+require_relative 'min_tree.rb'
+require_relative 'node.rb'
+
+RSpec.describe MinTree, type: Class do 
+
+	let (:root) { Node.new("The Matrix", 87) }
+
+  let (:tree) { MinTree.new(root) }
+  let (:pacific_rim) { Node.new("Pacific Rim", 72) }
+  let (:braveheart) { Node.new("Braveheart", 78) }
+  let (:jedi) { Node.new("Star Wars: Return of the Jedi", 80) }
+  let (:donnie) { Node.new("Donnie Darko", 85) }
+  let (:inception) { Node.new("Inception", 86) }
+  let (:district) { Node.new("District 9", 90) }
+  let (:shawshank) { Node.new("The Shawshank Redemption", 91) }
+  let (:martian) { Node.new("The Martian", 92) }
+  let (:hope) { Node.new("Star Wars: A New Hope", 93) }
+  let (:empire) { Node.new("Star Wars: The Empire Strikes Back", 94) }
+  let (:mad_max_2) { Node.new("Mad Max 2: The Road Warrior", 98) }
+
+	it "is a class" do 
+		expect(MinTree.new(:jedi)).to_not be(nil)
+	end
+
```

Comment:
- [ ] You have extra, unnecessary whitespace here (I'm only adding comments like this to emulate what a code review would be like in the "real world").